Source: Upstream
Upstream: Yes
Reason: Test Failure with - in path
http://redmine.ruby-lang.org/issues/show/883

Index: lib/rdoc/markup/to_html_crossref.rb
===================================================================
--- a/lib/rdoc/markup/to_html_crossref.rb	(revision 22822)
+++ b/lib/rdoc/markup/to_html_crossref.rb	(working copy)
@@ -66,5 +66,5 @@ class RDoc::Markup::ToHtmlCrossref < RDo
                       # one special character (period, slash, or
                       # underscore).
-                      | [\/\w]+[_\/\.][\w\/\.]+
+                      | (?:\.\.\/)*[-\/\w]+[_\/\.][-\w\/\.]+
 
                       # Things that have markup suppressed

